README.md

```diff
@@ -128,17 +148,222 @@ To start Parity Ethereum as a regular user using `systemd` init:
 2. Copy release to bin folder, write `sudo install ./target/release/parity /usr/bin/parity`
 3. To configure Parity Ethereum, write a `/etc/parity/config.toml` config file, see [Configuring Parity Ethereum](https://paritytech.github.io/wiki/Configuring-Parity) for details.
 
-## Parity Ethereum toolchain
+## 4. Documentation <a id="chapter-004"></a>
+
+Official website: https://parity.io
+
+Be sure to [check out our wiki](https://wiki.parity.io) for more information.
+
+### Viewing documentation for Parity Ethereum packages
+
+You can generate documentation for a Parity Ethereum Rust package and have it automatically open in your web browser using [rustdoc with Cargo](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html#using-rustdoc-with-cargo), (of the The Rustdoc Book), by running the the following commands:
+
+```
+cargo doc --package <spec> --open
```

Comment:
I think the default instruction should be to build **all** docs right?

Reply:
Yes thanks. I've addressed this in commit 95dd2cb2bb811e3073afe144ca389a90bda0411f